core/controllers/profile.py

```diff
@@ -83,6 +86,9 @@ def get(self, username):
                 utils.get_time_in_millisecs(user_settings.first_contribution_datetime)
                 if user_settings.first_contribution_datetime else None),
             'profile_picture_data_url': user_settings.profile_picture_data_url,
+            'created_explorations_count': len(user_contributions.created_explorations),
```

Comment:
Can you check with @tdtshafer what data he needs for the frontend? I suspect it's more than just the count...

Do you also intend to display private and/or deleted explorations? If not, you'll need to filter them out before they get to the frontend.